src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -93,14 +95,14 @@ private void handleDeletePerson() {
      */
     @FXML
     private void handleNewPerson() {
-        Optional<Person> newPerson = Optional.of(new Person());
+        Optional<ReadablePerson> prevInput = Optional.of(new Person());
```

Comment:
I'm also using this variable to store the previous input data to repopulate the dialog if the dialog previously closed with a warning. How does `latestPersonData` sound?